.github/workflows/test.yml

```diff
@@ -6,10 +6,7 @@ jobs:
   test:
     strategy:
       matrix:
-        ruby: [2.5, 2.6, 2.7]
-        include:
-          - ruby: 3.0
-            experimental: true
+        ruby: [2.5, 2.6, 2.7, 3.0]
```

Comment:
I recommend quoting these Ruby versions. Without the quote YAML interprets the values as numbers, hence `3.0` will end up being `3` and the tests will end up running on Ruby v3.1 when that is released later this year.

Reply:
Thanks, that would have been a rude discovery later! 738b4f09e5a5aa1d2b730c5496cafcef21aaac05